websites/A/asset.party/metadata.json

```
@@ -0,0 +1,22 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.7",
+	"author": {
+		"id": "521762564062052393",
+		"name": "Dwifte#0661"
```

Discriminator added incorrectly by old version of `pmd`
```suggestion
		"name": "Dwifte"
```